src/app/pages/send/send-crypto-asset-form/components/recipient-accounts-drawer/account-list-item.tsx

```diff
@@ -16,41 +16,47 @@ import { StacksAccount } from '@app/store/accounts/blockchain/stacks/stacks-acco
 interface AccountListItemProps {
   account: StacksAccount;
   onClose(): void;
+  showBtcAddress?: boolean;
+  showStxAddress?: boolean;
 }
-export const AccountListItem = memo(({ account, onClose }: AccountListItemProps) => {
-  const { setFieldValue, values } = useFormikContext<
-    BitcoinSendFormValues | StacksSendFormValues
-  >();
-  const [component, bind] = usePressable(true);
-  const name = useAccountDisplayName(account);
+export const AccountListItem = memo(
+  ({ account, onClose, showBtcAddress, showStxAddress }: AccountListItemProps) => {
+    const { setFieldValue, values } = useFormikContext<
+      BitcoinSendFormValues | StacksSendFormValues
+    >();
+    const [component, bind] = usePressable(true);
+    const name = useAccountDisplayName(account);
 
-  const btcAddress = useBtcNativeSegwitAccountIndexAddressIndexZero(account.index);
+    const btcAddress = useBtcNativeSegwitAccountIndexAddressIndexZero(account.index);
 
-  const onSelectAccount = () => {
-    const isBitcoin = values.symbol === 'BTC';
-    setFieldValue('recipient', isBitcoin ? btcAddress : account.address);
-    setFieldValue('recipientAddressOrBnsName', isBitcoin ? btcAddress : account.address);
-    onClose();
-  };
+    const onSelectAccount = () => {
+      const isBitcoin = values.symbol === 'BTC';
+      setFieldValue('recipient', isBitcoin ? btcAddress : account.address);
+      setFieldValue('recipientAddressOrBnsName', isBitcoin ? btcAddress : account.address);
+      onClose();
+    };
 
-  return (
-    <AccountListItemLayout
-      account={account}
-      accountName={<AccountName account={account} />}
-      avatar={
-        <AccountAvatarItem index={account.index} publicKey={account.stxPublicKey} name={name} />
-      }
-      balanceLabel={<AccountBalanceLabel address={account.address} />}
-      btcAddress={btcAddress}
-      index={account.index}
-      isActive={false}
-      isLoading={false}
-      mt="loose"
-      onSelectAccount={onSelectAccount}
-      stxAddress={account.address}
-      {...bind}
-    >
-      {component}
-    </AccountListItemLayout>
-  );
-});
+    return (
+      <AccountListItemLayout
+        account={account}
+        accountName={<AccountName account={account} />}
+        avatar={
+          <AccountAvatarItem index={account.index} publicKey={account.stxPublicKey} name={name} />
+        }
+        balanceLabel={<AccountBalanceLabel address={account.address} />}
+        btcAddress={btcAddress}
+        index={account.index}
+        isActive={false}
+        isLoading={false}
+        mt="loose"
+        onSelectAccount={onSelectAccount}
+        stxAddress={account.address}
+        showBtcAddress={showBtcAddress}
+        showStxAddress={showStxAddress}
+        {...bind}
```

Comment:
Try removing the bind temporarily and you'll see type errors that are hidden because of it

Reply:
removed these props at all